traveler-swift-core/TravelerKit/Models/CancellationError.swift

```diff
@@ -11,4 +11,6 @@ import Foundation
 public enum CancellationError: Error {
     case expiredQuote
     case notCancellable
+    case reasonNotExist
```

Comment:
Better naming here:

`reasonRequired`

Reply:
I created `reasonNotExist` to indicate that the reason selected by the user is not on the list of supported reasons provided by the provider.

Reply:
I'm thinking about if we need this error. On the other hand,  missing a reason when canceling an order is not possible since the cancellation request requires a non-optional reason string when initializing.

Reply:
In my opinion error cases should only cover errors that could happen. If there's no scenario where an unsupported reason can be created then there's no need for this case